en/simulation/hitl.md

```diff
@@ -1,36 +1,28 @@
 # Hardware in the Loop Simulation \(HITL\)
 
-Hardware-in-the-Loop (HITL or HIL) is a simulation mode in which normal PX4 firmware is run on real flight controller hardware. 
+Hardware-in-the-Loop (HITL or HIL) is a simulation mode in which normal PX4 firmware is run on real flight controller hardware.
 This approach has the benefit of testing most of the actual flight code on the real hardware.
 
-PX4 supports HITL for multicopters (using jMAVSim or Gazebo) and fixed wing (using Gazebo or X-Plane demo/full version). 
+PX4 supports HITL for multicopters (using jMAVSim or Gazebo) and fixed wing (using Gazebo).
 
 
 ## HITL-Compatible Airframes {#compatible_airframe}
 
 The current set of compatible airframes vs Simulators is:
 
-Airframe | `SYS_AUTOSTART` | X-Plane | Gazebo | jMAVSim
+Airframe | `SYS_AUTOSTART` | Gazebo | jMAVSim
```

Comment:
@julianoes - what frames can be used for Gazebo Fixed-Wing simulation. There were none listed here previously (only HILStar for XPlane).

Reply:
I don't think one is compatible. Last time I tried it was messy and the mixers were not correct.

Reply:
@julianoes OK, so that means we no longer have a Fixed wing HIL simulator right?

Reply:
I guess so. At least none that is maintained.

Reply:
Thanks. Not ideal.